src/vivarium_gates_mncnh/components/mortality.py

```diff
@@ -258,7 +258,18 @@ def on_time_step(self, event: Event) -> None:
 
         pop = self.population_view.get(event.index)
         alive_idx = pop.index[pop[COLUMNS.CHILD_ALIVE] == "alive"]
-        mortality_rates = self.death_in_age_group(alive_idx)
+        mortality_rates1 = self.death_in_age_group(alive_idx)
+        mortality_rates2 = self.death_in_age_group.source(alive_idx)
+        
+        # I can't figure out where this pipeline is being modified, but I think there is something wrong with the modification
+        import numpy as np
+        if not np.allclose(mortality_rates1, mortality_rates2):
```

Comment:
These will be different once we add interventions to the model, but at this point I expect them to be the same.